scss/_patterns_code-snippet.scss

```diff
@@ -7,6 +7,10 @@ $code-snippet-header-v-spacing: $spv-inner--x-small--scaleable;
   .p-code-snippet {
     margin-bottom: $spv-outer--scaleable;
 
+    &.is-bordered {
+      border: 1px solid $color-mid-x-light;
```

Comment:
@lyubomir-popov Should it be one of theme colors? We have `$colors--light-theme--border-default`,  `-border-high-contrast`, `-border-low-contrast`.

Reply:
-border-low-contrast please

Reply:
`-border-low-contrast` looks pretty close to `$color-mid-x-light`, go with that?